app/[locale]/10years/_components/TenYearHero.tsx

```diff
@@ -0,0 +1,145 @@
+"use client"
+
+import { useEffect, useState } from "react"
+
+import { Image } from "@/components/Image"
+import ParallaxImage from "@/components/Image/ParallaxImage"
+
+import TenYearBackgroundImage from "@/public/images/10-year-anniversary/10-year-background.png"
+import TenYearGraphicImage from "@/public/images/10-year-anniversary/10-year-graphic.png"
+
+const TenYearHero = () => {
+  const [words, setWords] = useState<{ text: string; words: string[] }>({
+    text: "censorship resistance",
+    words: [
+      "censorship resistance",
+      "100% uptime",
+      "decentralization",
+      "community building",
+      "developer growth",
+      "global collaboration",
+      "cypherpunk values",
+      "hackathons",
+      "censorship resistance",
+      "permissionless finance",
+      "credible neutrality",
+      "the infinite garden",
+      "client diversity",
+    ],
+  })
+
+  // loops over chars to morph a text to another
+  const morpher = (start: string, end: string): void => {
```

Comment:
DRY 😢 

nice to have: refactor `Morpher` to accept `words`, and `chars` (as optional) as props and reuse logic here.